lib/camera_screen.dart

```diff
@@ -55,39 +52,34 @@ class _CameraScreenState extends State<CameraScreen> {
         margin: const EdgeInsets.fromLTRB(0, 35, 0, 20),
         child: Column(
           children: [
-            Stack(
-              alignment: Alignment.center, 
-              children: const [
-                Positioned(left: 10, child: BackButton()),
-                Align(
+            Stack(alignment: Alignment.center, children: const [
+              Positioned(left: 10, child: BackButton()),
+              Align(
                   child: Text(
-                    "From Camera",
-                    textAlign: TextAlign.center,
-                    textScaleFactor: 1.5,
-                  )
-                ),
-              ]
-            ),
+                "From Camera",
+                textAlign: TextAlign.center,
+                textScaleFactor: 1.5,
+              )),
+            ]),
             const SizedBox(height: 20),
             getCameraPreview(screenWidth, screenHeight),
             const SizedBox(height: 10),
             ElevatedButton(
               style: ElevatedButton.styleFrom(
                 fixedSize: const Size(50, 50),
-                shape: const CircleBorder(side: BorderSide(color: Colors.black)),
-                backgroundColor: Colors.grey.shade600,
-                foregroundColor: Colors.grey.shade300,
+                shape:
+                    const CircleBorder(side: BorderSide(color: Colors.black)),
+
```

Comment:
Again, you're missing these 2 lines for the color of the shutter button. Just make sure to revert the deleted lines in the merge conflicts.